setup.cfg

```diff
@@ -1,56 +1,7 @@
-[metadata]
-name = resqpy
-# version = 0.1.0
-# author = Example Author
-# author_email = author@example.com
-description = Python API for working with RESQML files
-long_description = file: README.md
-long_description_content_type = text/markdown
-url = https://github.com/bp/resqpy
-classifiers =
-    License :: OSI Approved :: MIT License
-    Operating System :: OS Independent
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-    Development Status :: 4 - Beta
-    # Development Status :: 5 - Production/Stable
-    Topic :: Scientific/Engineering
-    Topic :: System :: Filesystems
-    Topic :: Scientific/Engineering :: Information Analysis
-
-[options]
-packages = find:
-include_package_data = True
-python_requires = >=3.7, <3.10
-setup_requires = setuptools_scm[toml]
-install_requires = 
-    numpy
-    pandas
-    h5py
-    lxml
-    lasio
-
 [options.package_data]
 # Include resqpy/olio/data/*.json
 * = data/*.json
```

Comment:
I believe this section is also setuptools-specific, so can be removed / migrated. The equivalent in poetry I think is the "Include" section:

https://python-poetry.org/docs/pyproject/#include-and-exclude